main.js

```diff
@@ -0,0 +1,95 @@
+
+let w = 40;
+let h = 30;
+let world = Array(h);
+for(let i = 0; i < world.length; i ++){
+  world[i] = Array(w);
+  world[i].fill("[ ]");
+}
+// seed
+world[(Math.floor(h/2))][(Math.floor(w/2))] = "[0]";
```

Comment:
Initial state function maybe? Seems you're putting some "alive cells" in the middle of the board.